### DISCUSSION TITLE
Error "Invalid network, net_version is: 5" when trying to use the ETH testnet configs

### DISCUSSION THREAD

**Discussion by RoniFinTech** [Web3Auth/Web3Auth#945] [Help]
Took the network configs from here:
https://web3auth.io/docs/connect-blockchain/ethereum/web#testnet

Any thought?

**Comment by Saviour1001** [comment]
Here are [examples](https://github.com/Web3Auth/examples/tree/main/web-modal-sdk/evm) created by us for helping you setup quickly. Changing the `chainId` and `rpcTarget` is enough for shifting from Eth Mainnet to testnet.
